### ISSUE TITLE
SyntaxError: Unexpected end of JSON' input on load

### ISSUE THREAD

**Issue by stridera** [kavod-io/vscode-jest-test-adapter#141]
**Things to consider before raising a bug**
This extension needs to understand how your project(s) are structured, where to find your test configuration and test files among other things.  It turns out that there are many ways to structure JS projects and the extension may not currently accommodate your project.  The following section is to aid in collecting information to identify the issue.

**Describe the bug**
- [ ] test load missing suites/tests
- [ ] test load errors
- [ ] test running errors

**To Reproduce**

**Expected behavior**

**Screenshots**

**Link to source**
Is there an OSS project demo or prod exposing this problem.

**Desktop (please complete the following information):**
 - OS: [e.g. Windows, OSX]

**Additional context**

**Comment by luispresuelgit** [comment]
Hi there @stridera , were you able to solve this issue? Running into something similar

**Comment by stridera** [comment]
Nope.  I just stuck with the command line.  It doesn't seem like it's maintained, so I had no expectations.
